metadata-ingestion/src/datahub/ingestion/source/superset.py

```diff
@@ -352,9 +362,13 @@ def emit_chart_mces(self) -> Iterable[MetadataWorkUnit]:
 
         while current_chart_page * PAGE_SIZE <= total_charts:
             chart_response = self.session.get(
-                f"{self.config.connect_uri}/api/v1/chart",
+                f"{self.config.connect_uri}/api/v1/chart/",
                 params=f"q=(page:{current_chart_page},page_size:{PAGE_SIZE})",
             )
+            if chart_response.status_code != 200:
+                logger.warning(f"Failed to get chart data: {chart_response.text}")
+            chart_response.raise_for_status()
```

Comment:
will this break entire connector?

Reply:
(or just skip the chart)

Reply:
This is the call to get the list of charts. It still breaks the entire connector, as was the behavior before, but just makes the error message more clear